src/content/platform-overview/cluster-management/cluster-api/architecture/index.md

```diff
@@ -32,12 +32,37 @@ By default, the machines are split into three different failure domains or zones
 There is a machine created as a bastion that helps us with the operations. It is the single entry point to the running infrastructure. This way all the cluster machines can live in a private network and expose the services running on them via explicit configuration.
 
 {{< tabs >}}
-{{< tab id="flags-openstack" title="OpenStack">}}
+{{< tab id="flags-openstack" title="OpenStack (CAPO)">}}
```

Comment:
Let's use shortcodes so that we have all provider names in one central place. Please have a look at

- Examples like `{{< tab id="cluster-vintage-azure" for-impl="vintage_azure" >}}`
- `src/config.yaml` as source of truth. If you want to change to `Cloud (provider)` order – I'm fine with it since we discussed that it reads better – please change it here.

There are surely many `{{< tab ... >}}` locations not fixed yet, such as the ones here. So let's do that. But you don't need to include that into the scope of this PR if it's too much effort.

Reply:
Thanks, I'll move this to another PR so I can merge this